raiden/network/blockchain_service.py

```diff
@@ -111,6 +113,16 @@ def token(self, token_address: bytes) -> Token:
 
         return self.address_to_token[token_address]
 
+    def channel_manager(self, channel_manager_address):
```

Comment:
Do you need `channel_manager` because I don't see it being used in the PRs so far or is it just for completeness?

Reply:
Yes, it will be used in an upcomming PR